tests/kernel/run_ltp.pm

```diff
@@ -348,7 +348,11 @@ sub run {
 
     assert_script_run('cd $LTPROOT/testcases/bin');
 
-    my $ver_linux   = script_output('$LTPROOT/ver_linux');
+    my $ver_linux = '$LTPROOT/ver_linux';
+    my $log       = '/tmp/ver_linux_before.txt';
+    script_run("$ver_linux > $log 2>&1");
+    upload_logs($log);
+    my $out         = script_output("cat $log");
```

Comment:
out - is too broad name . What about `$ver_linux_out` ?

Reply:
IMHO it's fine.

Reply:
Hm, ok. After reviewing how it's used I agree with @asmorodskyi :)

Reply:
@asmorodskyi @okurz `$log` is also broad name :-). I changed all 3 variables (I also don't like broad names, but `$ver_linux_out`, `$ver_linux_log`, `$ver_linux_path` looked quite long to me).